### PR TITLE
[ENG-1328] Cleans up welcome page

### PR DESCRIPTION
**Pull request by vsreekanti**
## Describe your changes and why you are making these changes

As discussed on Linear, this PR makes a number of small fixes to the welcome page:
* Makes all the blocks even width. 
* Removes the light grey background adds dark blue borders to each block, combined with a light blue hover background.
* Adds Aqueduct logo to the top of the page.
* Cleans up getting started instructions to match the latest information.
* Animates wave emoji on page load and when hovering over the welcome block.

https://user-images.githubusercontent.com/867892/184013864-69cd5881-51b0-4361-ba63-4f191d42b150.mov

## Related issue number (if any)

ENG-1328

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [N/A] If this is a new feature, I have added unit tests and integration tests.
- [N/A] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

**Comment by likawind** [comment]
This is very helpful @vsreekanti ! I think the only thing I'm unsure about is if we want to put integration connection as the first step, esp. that majority of step 2 and 3 are actually both using the demo. I'm considering the following change:
```
1. (current step 2) Create your first workflow using our demo database - see our quick start guide for an example.
2. (current step 3)
3. Try out with your own data by <link> connecting to an integration. </link>
```
My motivation is this is easier to follow for first-time user. And for non-first-time users it's obvious for them to ignore step 1 and 2 since they know they've already done it.

File: src/ui/common/src/components/cards/GettingStartedTutorial.tsx [hunk 1]
```diff
@@ -47,6 +44,34 @@ const GettingStartedTutorial: React.FC<GettingStartedTutorialProps> = ({
     greeting = 'Welcome to Aqueduct!';
   }
 
+  const waveEmoji = () => {
```
Comment (agiron123) [inline src/ui/common/src/components/cards/GettingStartedTutorial.tsx:47]: This is fun! 😍

**Comment by agiron123** [comment]
> This is very helpful @vsreekanti ! I think the only thing I'm unsure about is if we want to put integration connection as the first step, esp. that majority of step 2 and 3 are actually both using the demo. I'm considering the following change:
> 
> ```
> 1. (current step 2) Create your first workflow using our demo database - see our quick start guide for an example.
> 2. (current step 3) Go to Workflows page to see visualization of the workflow you just created
> 3. Try out with your own data by <link> connecting to an integration. </link>
> ```
> 
> My motivation is this is easier to follow for first-time user. We could minimize the risk that one user decide to connect to an integration, and the following steps becomes less helpful. And for non-first-time users it's obvious for them to ignore step 1 and 2 since they know they've already done it.
> 
> I'm curious about how other folks @agiron123 feel as well

@likawind these are great ideas, but are a bit out of scope for the task at hand. I've documented these and created a new design task here for us to follow up:
https://linear.app/aqueducthq/issue/ENG-1562/improve-design-of-first-time-user-experience

I definitely agree that there is a lot that we can do to make our getting started page easier to follow. However, I think this warrants a bigger design discussion on how we wish to surface our features to users.

**Review by agiron123 — APPROVED** [review]

**Comment by vsreekanti** [comment]
Thanks for creating the new task, @agiron123! That makes sense to me. Anything you'd like me to change before we merge this?

**Pull request merged**